gnowsys-ndf/gnowsys_ndf/ndf/urls/__init__.py

```diff
@@ -45,6 +45,7 @@
     (r'^(?P<group_id>[^/]+)/video', include('gnowsys_ndf.ndf.urls.video')),
     (r'^(?P<group_id>[^/]+)/page', include('gnowsys_ndf.ndf.urls.page')),
     (r'^(?P<group_id>[^/]+)/group', include('gnowsys_ndf.ndf.urls.group')),
+    (r'^(?P<group_id>[^/]+)/ncert_partner', include('gnowsys_ndf.ndf.urls.partner')),
```

Comment:
Can we have `/partner` as in url pattern instead of `/ncert_partner`. Because GN wanted it as generic and for every site.